### PR TITLE
Remove LzoCodec

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: src/main/java/org/apache/hive/hcatalog/data/JsonSerDe.java [hunk 1]
```diff
@@ -18,18 +18,6 @@
  */
 package org.apache.hive.hcatalog.data;
 
-import java.io.IOException;
```
Comment (electrum) [inline src/main/java/org/apache/hive/hcatalog/data/JsonSerDe.java]: Revert this change

**Review by electrum — APPROVED** [review]

File: src/main/java/org/apache/hive/hcatalog/data/JsonSerDe.java (hunk 1 above)
Comment (dain) [reply src/main/java/org/apache/hive/hcatalog/data/JsonSerDe.java]: I have no idea how that happened.
